project/src/main/webapp/sps/data/Event.java

```diff
@@ -0,0 +1,96 @@
+import java.util.Date;
+import java.util.Random; 
+import java.util.Set;
+
+/** Class containing Event object 
+Setters for variables that user can change about event
+*/
+public class Event {
+  private final int eventId;
+  private String name;
+  private String description;
+  private Set<String> labels;
+  private String location;
+  private Date date;
+  private Set<VolunteeringOpportunity> opportunities;
+  private Set<User> attendees;
+  private User host;
+
+  Random rand = new Random(); 
```

Comment:
Is rand used anywhere else? if not you can set in constructor.

Reply:
my reasoning for setting it not in the constructor is that many Event objects will be created and I dont want to reinitialize rand each time

Reply:
Then I guess you should make it static final?